whois-api/src/main/java/net/ripe/db/whois/api/httpserver/dos/WhoisDoSFilter.java

```diff
@@ -81,9 +81,9 @@ public String getWhitelist() {
     public void setWhitelist(final String commaSeparatedList) {
         clearWhitelist();
         for (String address : StringUtil.csvSplit(commaSeparatedList)) {
-            addWhitelistAddress(address, false);
+            addWhitelistAddressWithoutLog(address);
```

Comment:
Since we are registering two DOS filter , is the whitelist separate for query and update? If I remove an IP from whitelist will it be removed in both for query and update ?

Reply:
White list will be created for update and for query (1 each)
If you remove something for the whiteList it will be removed from both.

However, I would like to use just one whiteList and not 1 each but I couldn't find the way

Reply:
We should be using one whitelist only.